src/backend/partaj/core/api/referral_lite.py

```diff
@@ -101,9 +101,9 @@ def list(self, request, *args, **kwargs):
         if len(unit):
             es_query_filters += [{"terms": {"units": unit}}]
 
-        user = form.cleaned_data.get("user")
```

Comment:
Should we not let this one in place and just add `users_unit_name` in addition to it?